modules/oc-mirror-v2-about.adoc

```diff
@@ -16,14 +16,15 @@ oc-mirror plugin v2 has the following features:
 * Verifies that the complete image set specified in the image set configuration file is mirrored to the mirrored registry, regardless of whether the images were previously mirrored or not.
 
 * Uses a cache system instead of metadata.
+// Does the cache about refer to the cache of files mentioned in Luigi's demo of the delete feature, where he says that the plugin uses a local cache to cache images in case of a failure? If so, I can expand this bullet point to explicitly talk about how the cache keeps images around in case of failure, preventing the need to start over.
```

Comment:
New question as a result of watching a demo

Reply:
Yes, good idea